catalog/validator/catalog-validator-metacardduplication/src/main/java/org/codice/ddf/validator/metacard/duplication/DuplicationValidator.java

```diff
@@ -113,119 +124,185 @@ public void setWarnOnDuplicateAttributes(String[] attributeStrings) {
     }
   }
 
+  /**
+   * Setter for the list of attributes to test for duplication in the local catalog. Resulting
+   * attributes will cause the metacard to be rejected.
+   *
+   * @param attributeStrings
+   */
+  public void setRejectOnDuplicateAttributes(String[] attributeStrings) {
```

Comment:
Think we need a getter for this as well

Reply:
There were no getters for the other `setOnDuplicateAttributes` so I did not add one just for the reject case

Reply:
That's strange 🤔 I am under the impression that that method is needed to populate the admin ui with the up to date values